man/org.freedesktop.login1.xml

```diff
@@ -525,8 +546,19 @@ node /org/freedesktop/login1 {
       using an RTC timer and hibernated. The only argument is the polkit interactivity boolean
       <varname>interactive</varname> (see below). The main purpose of these calls is that they enforce
       polkit policy and hence allow powering off/rebooting/suspending/hibernating even by unprivileged
-      users. They also enforce inhibition locks. UIs should expose these calls as the primary mechanism to
-      poweroff/reboot/suspend/hibernate the machine.</para>
+      users. They also enforce inhibition locks for non-privileged users. UIs should expose these calls
+      as the primary mechanism to poweroff/reboot/suspend/hibernate the machine. Methods
+      <function>PowerOffWithFlags()</function>, <function>RebootWithFlags()</function>,
+      <function>HaltWithFlags()</function>, <function>SuspendWithFlags()</function>,
+      <function>HibernateWithFlags()</function>, <function>HybridSleepWithFlags()</function> and
+      <function>SuspendThenHibernateWithFlags()</function> adds uint64 flags to allow for
```

Comment:
uint64 is C jargon. I don't think we need to show C syntax at all. Also adds→add. Maybe
> … add `<varname>flags</varname>` parameter. 
> When the `<varname>flags</varname>` is 0 then these methods behave just like the versions without flags.
> When `<constant>SD_LOGIND_ROOT_CHECK_INHIBITORS</constant>` (0x01) is set, active inhibitors are honoured for privileged users too.